lib/models/device.coffee

```diff
@@ -88,9 +92,33 @@ getDeviceModel = (deps, opts) ->
 		if semver.lt(version, minVersion)
 			throw new Error("Incompatible supervisor version: #{version} - must be >= #{minVersion}")
 
-	addApplicationName = (device) ->
+	###*
+	# @summary Get Dashboard URL for a specific device
+	# @function getDashboardUrl
+	#
+	# @param {Object} options - options
+	# @param {Number} options.appId - Application id
+	# @param {Number} options.deviceId - Device id
+	#
+	# @returns {String} - Dashboard URL for the specific device
+	# @throws Exception if either appId or deviceId are empty
+	#
+	# @example
+	# dashboardDeviceUrl = resin.models.device.getDashboardUrl({ appId: 123, deviceId: 456 })
+	###
+	exports.getDashboardUrl = getDashboardUrl = (options = {}) ->
+		isFinite = require('lodash/isFinite')
+		url = require('url')
```

Comment:
Nit: we normally do requires at the top level in the SDK, it'd be good to keep everything together.

You're right that we do this in the CLI though, and I'm now wondering whether we should in the SDK for startup performance too... Not right now though :-).